tests/fast-integration/monitoring/grafana.js

```diff
@@ -0,0 +1,209 @@
+const {
+    assert,
+    expect,
+} = require("chai");
+
+const {
+    getEnvOrDefault,
+    getVirtualService,
+    toBase64,
+    k8sApply,
+    k8sDelete,
+    patchDeployment,
+    k8sAppsApi,
+    retryPromise,
+    sleep,
+    waitForDeployment,
+    waitForPodWithLabel,
+} = require("../utils");
+
+const { queryGrafana } = require("./client");
+
+async function assertPodsExist() {
+    let namespace = "kyma-system";
+    await waitForPodWithLabel("app", "grafana", namespace);
+}
+
+async function assertGrafanaRedirectsExist() {
+    if (getEnvOrDefault("KYMA_MAJOR_VERSION", "2") === "2") {
+        await checkGrafanaRedirectsInKyma2();
+    } else {
+        await checkGrafanaRedirectsInKyma1();
+    }
+}
```

Comment:
I am wondering if we have ever run those tests for Kyma 1. @rakesh-garimella, do you know. If it's not relevant. let us get rid of it.

Reply:
Would like to do that then in the separate clean-up PR I'm preparing. This one is mostly about restructuring and including tests in the default npm test

Reply:
yeah we should handle it in a separate PR as we should also cleanup from various pipelines where it is set.